src/Models/Board.js

```diff
@@ -143,33 +143,100 @@ export default class Board{
+            console.log(y)
+            this.grid[x][y].Occupant = new Monkey(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++;
+            this.grid[x][y].Occupant = new Elephant(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++;
+            this.grid[x][y].Occupant = new Lion(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++;
+            this.grid[x][y].Occupant = new Gorilla(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++;
+            this.grid[x][y].Occupant = new Elephant(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++;
+            this.grid[x][y].Occupant = new Monkey(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++;
+            this.grid[x][y].Occupant = new Cheetah(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x++; 
+        }
+        else if(y == this.ROW_SIZE - 1){
+            this.grid[x][y].Occupant = new Cheetah(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Monkey(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Elephant(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Lion(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Gorilla(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Elephant(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Monkey(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+            this.grid[x][y].Occupant = new Cheetah(x,y);
+            this.grid[x][y].Occupant.Owner = player_;
+            x--; 
+        }
     }
     /**
      * Creates a Grid of [ROW_SIZE] by [COL_SIZE] Tiles
      */
     createNewGrid(){
-        let occupant;
+        // initialize all tiles to null
         for(var x=0;x<this.COL_SIZE;x++){
             let row = [];
             for(var y=0;y<this.ROW_SIZE;y++){
-                if(y < 2){
-                    occupant = this.generateRandomPawn(x,y);
-                    occupant.Owner = Players.ONE;
-                }
-                else if(y >this.ROW_SIZE-3 || y < 2){
-                    occupant = this.generateRandomPawn(x,y);
-                    occupant.Owner = Players.TWO;
-                }
-                else
-                    occupant = null;
-
-                row[y]=new Tile(x,y,this.TILE_WIDTH,occupant);
+                row[y]=new Tile(x,y,this.TILE_WIDTH,null);
             }
             this.grid.push(row);
         }
+        // generate pieces for top player
+        y = 0;
```

Comment:
Try to incorporate this within the loop. Use if statements. Ok if they're only executed once. I prefer cleanliness and reduced hard coded variables.